### PR TITLE
Added tutorial for using react native on bitrise

### PR DESCRIPTION
**Pull request by kokomo88**

### CODE REVIEW

File: source/tutorials/building-react-native-projects-on-bitrise.html.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    brew install iojs
+    brew install --HEAD watchman
+    brew install flow
+
+In order to build a [React Native](https://facebook.github.io/react-native/) app you have to install [React Native](https://facebook.github.io/react-native/) with npm. [Node](https://nodejs.org/) is already installed on our VMs, so you don't have to do it manually.
+Add the [Bash Script Runner](https://github.com/bitrise-io/steps-bash-script) step to your workflow. Make sure you cd into your project's directory before running the npm install like in the following code:
+
+    #!/bin/bash
+    cd "./projectDir"
+    npm install react-native
```
Comment (viktorbenei) [inline source/tutorials/building-react-native-projects-on-bitrise.html.md]: you don't have to run one more command? something what converts/updates/creates the Xcode project?

File: source/tutorials/building-react-native-projects-on-bitrise.html.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+React Native recommends to install [iojs](https://iojs.org/en/index.html), [watchman](https://github.com/facebook/watchman), and [flow](http://flowtype.org/).
+
+These frameworks are not required to build a [React Native](https://facebook.github.io/react-native/) project, so if you don't use them, you can skip this part.
+Add the [Bash Script Runner](https://github.com/bitrise-io/steps-bash-script) step to your workflow.
+
+    #!/bin/bash
+    brew unlink node
+    brew uninstall node
+    brew install iojs
```
Comment (viktorbenei) [inline source/tutorials/building-react-native-projects-on-bitrise.html.md]: do you need `ions` for React Native, or only for `watchman` or for `flow`?

Comment (kokomo88) [inline source/tutorials/building-react-native-projects-on-bitrise.html.md]: Test project link : https://github.com/kokomo88/viccesteszt

2015-08-14 7:46 GMT+02:00 Viktor Benei <notifications@github.com>:

> In source/tutorials/building-react-native-projects-on-bitrise.html.md
> <https://github.com/bitrise-io/devcenter/pull/37#discussion_r37051104>:
>
> > +title: Building React Native projects on Bitrise
> > +---
> > +# Building React Native projects on Bitrise
> > +
> > +This tutorial shows you how to setup [React Native](https://facebook.github.io/react-native/) on [Bitrise](http://bitrise.io/).
> > +
> > +
> > +React Native recommends to install [iojs](https://iojs.org/en/index.html), [watchman](https://github.com/facebook/watchman), and [flow](http://flowtype.org/).
> > +
> > +These frameworks are not required to build a [React Native](https://facebook.github.io/react-native/) project, so if you don't use them, you can skip this part.
> > +Add the [Bash Script Runner](https://github.com/bitrise-io/steps-bash-script) step to your workflow.
> > +
> > +    #!/bin/bash
> > +    brew unlink node
> > +    brew uninstall node
> > +    brew install iojs
>
> do you need ions for React Native, or only for watchman or for flow?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bitrise-io/devcenter/pull/37/files#r37051104>.
>



-- 
Üdv, Kiss Zoltán

Comment (viktorbenei) [inline source/tutorials/building-react-native-projects-on-bitrise.html.md]: Thanks!

**Comment by viktorbenei** [comment]
Thanks for the submission! ;)

**Pull request merged**